deploy/operator/operator.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
Maybe I'm missing something here, could you please elaborate why we need to run the operator as DaemonSet and not as Deployment?

Reply:
kubernetes-nmstate is distributed, we did it by design, reason is we have to reduce the number of remote access so we can do node's networking rollback if we loss kubeapi connectivity after a bad network configuration.

Reply:
But Yossi is talking about the operator, not handler. With handler, we should deloy as a Deployment. In fact we have to, CSV does not support DaemonSets.

Reply:
Ohh my mistake, yep operator has to be a Daemon and operand at DaemonSet.